docs/content/guides/reverse-proxy.md

```diff
@@ -114,3 +114,25 @@ Here is an example config snippet:
   Include /etc/letsencrypt/options-ssl-apache.conf
 </VirtualHost>
 ```
+### Traefik
+
+In traefik we have to add the traefik labels and it is also very important: add our domain to "CMD_DOMAIN", "CMD_URL_ADDPORT=false", "CMD_PROTOCOL_USESSL=true".
```

Comment:
Which traefik labels? All of them? Please write down the absolute necessary labels that are needed to make hedgedoc work with hedgedoc.